lib/index.js

```diff
@@ -30,8 +30,16 @@ var Iterable = module.exports = integration('Iterable')
  */
 
 Iterable.prototype.track = function(payload, fn){
+  var endpoint;
+  if (payload["isUpdateCart"]) {
```

Comment:
it will have either isUpdateCart=true, isPurchase=true, or neither.

Also that's harder to read IMO